test/fixtures/docker-compose/.env

```diff
@@ -0,0 +1,4 @@
+DOCKER_IMG_TAG=:2.0
```

Comment:
Can you change this to 2.1, our methodology was wrong for our 2 digit version for 2.next